tests/playwright/src/model/pages/build-image-page.ts

```
@@ -170,4 +175,36 @@ export class BuildImagePage extends BasePage {
       }),
     );
   }
+
+  private async fillBuildImageForm(
+    imageName: string,
+    containerFilePath: string,
+    contextDirectory: string,
+    archType: string[] = [ArchitectureType.Default],
+  ): Promise<void> {
+    if (!containerFilePath) {
+      throw Error(`Path to containerfile is incorrect or not provided!`);
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)